docs/reference/models/index.md

```diff
@@ -43,6 +43,10 @@ Models for working with captions. These models represent the output of image cap
 
 Models for working with chat models. These models represent the input and output of chat models and models for OpenAI-compatible API.
 
+## [Visual Chat Models](./image_chat.md)
```

Comment:
It's a bit confusing that it's called Visual Chat Models and Image Chat models everywhere else.

Reply:
Fixed